api/v1/storagecluster_types.go

```diff
@@ -234,6 +236,14 @@ type EncryptionSpec struct {
 	KeyManagementService KeyManagementServiceSpec `json:"kms,omitempty"`
 }
 
+// MirroringSpec defines if data mirroring should be enabled for the StorageCluster.
+// It is optional and defaults to false.
+type MirroringSpec struct {
+	// +kubebuilder:default:=true
```

Comment:
Are we planning to enable DR by default in 4.9? AFAIK the plan is to enable DR as a day 2 operation.
@ShyamsundarR

Reply:
I'm confused. Didn't we want to enable mirroring by default?

Reply:
Set it to disabled by default.